src/frontend/komponenter/Fagsak/Saksoversikt/Saksoversikt.tsx

```diff
@@ -60,6 +60,13 @@ const StyledTabs = styled(Tabs)`
     margin-bottom: 1rem;
 `;
 
+const StyledAlert = styled(Alert)`
+    .navds-alert__wrapper {
+        display: block;
+        flex: 1;
+    }
+`;
```

Comment:
Denne trenger vi for å skape et mellomrom mellom tekst og "Se detaljer"-lenke. Den nye komponenten er stylet litt annerledes enn den forrige, så dette var endringen som måtte til for å få det til å funke som tidligere.

Reply:
Hvorfor tenger man flex 1 her?

Reply:
Uten den så funket ikke flex-boksen som er inni denne komponenten igjen 😬

Reply:
Hmm trodde flex atributten skulle brukes på objekter inne en flex-box 🤔
tror 
```
const StyledAlert = styled(Alert)\`
    .navds-alert__wrapper {
        display: flex;
    }
\`;
```
og 
```
const FlexSpaceBetween = styled.div\`
    display: flex;
    justify-content: space-between;
    width: 100%;
\`;
```
Skal funke
https://jsfiddle.net/2qf0st7d/8/

Eventuelt burde bare
```
const StyledAlert = styled(Alert)\`
    .navds-alert__wrapper {
        display: block;
    }
\`;
```
funke også
https://jsfiddle.net/2qf0st7d/7/

Reply:
Har prøvd begge de, men da ender den bare opp sånn her
![image](https://user-images.githubusercontent.com/25459913/179737007-d0bb31d0-e72c-4f58-8454-5c75f3eda8d8.png)

Reply:
Kan fjerne display: block, den trengs ikke egentlig, men flex: 1 trengs for å få det til å funke

Reply:
Mulig det funker fordi navds-alert__wrapper er et flex-item i seg selv, da den ligger inne i en flexbox den også 😄